airflow/executors/kubernetes_executor.py

```diff
@@ -633,12 +651,15 @@ def _labels_to_key(self, labels):
         return None
 
     def terminate(self):
+        """Termninates the watcher."""
         self.watcher_queue.join()
         self._manager.shutdown()
 
 
 class KubernetesExecutor(BaseExecutor, LoggingMixin):
+    """Executor for Kubernetes"""
     def __init__(self):
+        """Creates the object"""
```

Comment:
```suggestion
```
It is not necessary. We should keep the documentation in the class description

Reply:
Done